core/templates/components/forms/custom-forms-directives/thumbnail-display.component.html

```diff
@@ -0,0 +1,5 @@
+<img [src]="imageSourceInView "
```

Comment:
Drop trailing space before closing quote?

Reply:
This is a part of #11306.
See https://github.com/oppia/oppia/pull/11850#issuecomment-786942336 for details.